### DISCUSSION TITLE
Transitions without Tailwind is it possible?

### DISCUSSION THREAD

**Discussion by Dimonka2** [alpinejs/alpine#828] [1. Help] [answered]
I am planning to add Alpine to a bootstrap project and it seems that transitions are heavily bounded to Tailwind and there is no option to to use them in a "standalone" version. Is it true? Maybe there is a workaround?

**Comment by SimoTod** [comment, marked as answer]
They are not bound to tailwind, they just apply classes and styles. The example uses tailwind to demonstrate it but it's just an example.

**Reply by Dimonka2** [reply]
Example meaning, that I have to create similar css classes to tailwind in order to use it? Or maybe extract a transition css part from tailwind? What is the easiest way?

**Reply by SimoTod** [reply]
Yeah, that's correct, on and off. 
Alpine offers a custom preset, x-show.transition, that you can use out of the box or the `x-transition:*` directives to add more control.
The latter uses css classes that you need to create by yourself or include from an 3rd party stylesheet.
There are 3 directive types: start, during and end.
On x-transition:enter-start you want to define your starting point for the property/properties you want to animate, for example a class that gives `opacity: 0`. 
On during you then define the properties that your element should have during the whole transition: you usually want to give a class that tells your page to animate a specific property and the length of the animation, for example a class that gives `transition: opacity 1s;`. 
On end, you want to assign the final state, for example a class that gives `opacity: 1`. 

I hope it helps.
